src/sagemaker/lineage/action.py

```diff
@@ -104,13 +108,14 @@ def delete(self, disassociate=False):
                 source_arn=self.action_arn, sagemaker_session=self.sagemaker_session
             )
             _utils._disassociate(
-                destination_arn=self.action_arn, sagemaker_session=self.sagemaker_session
+                destination_arn=self.action_arn,
+                sagemaker_session=self.sagemaker_session,
             )
 
         self._invoke_api(self._boto_delete_method, self._boto_delete_members)
 
     @classmethod
-    def load(cls, action_name, sagemaker_session=None):
+    def load(cls, action_name: str, sagemaker_session: Session = None) -> object:
```

Comment:
can this return type be more specific? we are constructing an Action here.

Reply:
Sure, changed to Action object type